build-tools/jnienv-gen/Generator.cs

```diff
@@ -36,14 +48,21 @@ public static int Main (string [] args)
 					else
 						File.WriteAllText (jnienv_g_cs, content);
 				}
-				using (TextWriter w = new StringWriter ()) {
-					w.NewLine = "\n";
-					GenerateNativeLibSource (w);
-					string content = w.ToString ();
-					if (jnienv_g_c == "-" || jnienv_g_cs == "-")
-						Console.WriteLine (content);
-					else
-						File.WriteAllText (jnienv_g_c, content);
+				using (TextWriter sw = new StringWriter ()) {
+					using (TextWriter hw = new StringWriter ()) {
```

Comment:
You can "conventionally" save an indentation level by removing the `{` on the preceding line:

```csharp
using (TextWriter sw = new StringWriter ())
using (TextWriter hw = new StringWriter ()) {
    // …
}
```